commit 104a6c6e78baae0ad1da8ae297ee42c5fea080fe
Author: Sugizaki Yukimasa <i.can.speak.c.and.basic@gmail.com>
Date:   Thu Jan 4 23:58:06 2018 +0900

    vcsm: Define cache operation constants in user header
    
    Without this change, users have to use raw values (1, 2, 3) to specify
    cache operation.
    
    Signed-off-by: Sugizaki Yukimasa <i.can.speak.c.and.basic@gmail.com>

diff --git a/include/linux/broadcom/vmcs_sm_ioctl.h b/include/linux/broadcom/vmcs_sm_ioctl.h
index a4492b921b48..2395ff08c5ce 100644
--- a/include/linux/broadcom/vmcs_sm_ioctl.h
+++ b/include/linux/broadcom/vmcs_sm_ioctl.h
@@ -79,6 +79,11 @@ enum vmcs_sm_cache_e {
 	VMCS_SM_CACHE_BOTH,
 };
 
+/* Cache functions */
+#define VCSM_CACHE_OP_INV       0x01
+#define VCSM_CACHE_OP_CLEAN     0x02
+#define VCSM_CACHE_OP_FLUSH     0x03
+
 /* IOCTL Data structures */
 struct vmcs_sm_ioctl_alloc {
 	/* user -> kernel */
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1184_104a6c6e78baae0ad1da8ae297ee42c5fea080fe.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1184_104a6c6e78baae0ad1da8ae297ee42c5fea080fe.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
